Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/SpecialZone.hs

```diff
@@ -0,0 +1,33 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module Dashboard.ProviderPlatform.SpecialZone
+  ( module Dashboard.ProviderPlatform.SpecialZone,
+    module Reexport,
+  )
+where
+
+import Dashboard.Common as Reexport
+import Kernel.Prelude
+import Kernel.Storage.Esqueleto hiding (Point)
+
+data SpecialZoneEndpoint
```

Comment:
Why SpecialZone is part of ProviderPlatform? Isn't it outside of provider and rider platform?